docs/updating/from_4.0/to_4.1.md

```diff
@@ -0,0 +1,131 @@
+    ```
+
+After you update the settings, you can proceed to product remodeling.
```

Comment:
We don't use "remodeling" (or "product modeling") anywhere else, so maybe something else: adding your products (products and product types), populating your catalog...?